go/chat/inboxsource.go

```diff
@@ -281,7 +281,7 @@ func NewRemoteInboxSource(g *globals.Context, ri func() chat1.RemoteInterface) *
 	return s
 }
 
-func (s *RemoteInboxSource) Clear(ctx context.Context, uid gregor1.UID) error {
+func (s *RemoteInboxSource) Clear(ctx context.Context, uid gregor1.UID, opts types.ClearOpts) error {
```

Comment:
Should just have it take `nil` with an explicit default setup so it isn't so dependent on default values